inventory/multi_ec2.py

```diff
@@ -196,10 +197,36 @@ def get_inventory(self):
                     raise RuntimeError(result['err'])
                 else:
                     self.all_ec2_results[result['name']] = json.loads(result['out'])
+
+            # Check if user wants extra vars in yaml by
+            # having hostvars and all_group defined
+            for acc_config in self.config['accounts']:
+                if not acc_config.has_key('hostvars') and not acc_config.has_key('all_group'):
+                    continue
+
+                results = self.all_ec2_results[acc_config['name']]
+                # Verify the account results look sane
+                # by checking for these keys ('_meta' and 'hostvars' exist)
+                if not results.has_key('_meta') and not results['_meta'].has_key('hostvars'):
+                    continue
+
+               # Update each hostvar with the newly desired key: value
+                for host_property, value in acc_config['hostvars'].items():
+                    for data in results['_meta']['hostvars'].values():
```

Comment:
Instead of doing a continue above if it's missing _meta, could we instead put that if statement here?

That way we'd still get the group created, just nothing added to hostvars (because it's non-existent).